03-UnitsQuantities/Astropy_Units.ipynb

```diff
@@ -1413,18 +1231,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Defining the Quantity the simplest way without parenthesis.\n",
-    "%timeit big_array * u.m / u.s / u.kg / u.sr"
+    "((89.4 * u.ms) / (67.1 * u.us)).to(u.dimensionless_unscaled)"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# Defining the Quantity with the * operator and parenthesis.\n",
-    "%timeit big_array * (u.m / u.s / u.kg / u.sr)"
+    "# Science use case\n",
+    "\n",
+    "The information stored in the file `fake_sn_data.csv` is simulated data for the Type Ia supernova 2011fe. The only in the file are `JD_BEGIN` (time at the start of the exposure), `Rc` (magnitude in the Cousins R filter), and `EXPOSURE` (the exposure time for eah measurement).\n",
+    "\n",
+    "### What is the error in the code below?"
```

Comment:
Actually this might be a little confusing. Do we really need to throw in Table here? Can we instead showcase this "error" using just Python code?